### PR TITLE
Bugfix - misc bugfixes

### PR DESCRIPTION
**Pull request by yonikid15**
### Notes:
- Modal was overflowing outside screen. Might be caused by a clone update.
- ScrollToTop needed children prop in useEffect dependencies.
- Recursively update wordcounter if new skill button is triggered.

### CODE REVIEW

File: resources/assets/js/components/ApplicantSkills/SkillsWordCounter.tsx [hunk 1]
```diff
@@ -96,12 +96,17 @@ const addHardSkillButton: HTMLElement | null = document.getElementById(
 );
 
 const updateWordCounters = (): void => {
+  if (addSoftSkillButton) {
+    addSoftSkillButton.addEventListener("click", updateWordCounters);
```
Comment (Khristophor) [inline resources/assets/js/components/ApplicantSkills/SkillsWordCounter.tsx]: Do we have anything we can hook into to remove these event listeners?

Comment (yonikid15) [reply resources/assets/js/components/ApplicantSkills/SkillsWordCounter.tsx]: The add skill buttons are on the laravel side so I'm not sure how other than this.

Comment (tristan-orourke) [reply resources/assets/js/components/ApplicantSkills/SkillsWordCounter.tsx]: Yeah, in this case I'm not sure how else to do it.

Comment (Khristophor) [reply resources/assets/js/components/ApplicantSkills/SkillsWordCounter.tsx]: Alrighty, my only concern here is having a bunch of event listeners hanging around in the browser after heavy use, but we can keep an eye on it in the future.

**Pull request merged**